controller/api/public/stat_summary.go

```diff
@@ -426,15 +439,15 @@ func isInvalidServiceRequest(req *pb.StatSummaryRequest) bool {
 }
 
 func (s *grpcServer) queryProm(ctx context.Context, query string) (model.Vector, error) {
-	log.Debugf("Query request: %+v", query)
+	log.Debugf("Query request:\n %+v", query)
```

Comment:
Super tiny nit -- there's no need for the space after the line break here. As written it will indent the query text one character on the following line. Come to think of it though, maybe we _should_ indent the query, but in that case I'd indent by at least one tab:

```
	log.Debugf("Query request:\n\t%+v", query)
```

Same goes for the debug statement on line 450.

Reply:
oooh, yeah thanks!